packages/portal/spec-renderer-mini/src/components/OperationsListSectionHeader.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <button
+    :aria-controls="contentElementId"
+    :aria-expanded="isCollapsed ? 'false' : 'true'"
+    :aria-label="isCollapsed ? `Expand section &quot;${name}&quot;` : `Collapse section &quot;${name}&quot;`"
+    class="header"
+    type="button"
+    @click="emits('toggle')"
```

Comment:
Where you have this defined as `emits` can you change to `emit` to match the standard?

Reply:
Fixed!